src/core/settings.c

```diff
@@ -1423,6 +1440,9 @@ QuicSettingsDumpNew(
     if (Settings->IsSet.EncryptionOffloadAllowed) {
         QuicTraceLogVerbose(SettingEncryptionOffloadAllowed,        "[sett] EncryptionOffloadAllowed   = %hhu", Settings->EncryptionOffloadAllowed);
     }
+    if (Settings->IsSet.ReliableResetEnabled) {
+        QuicTraceLogVerbose(SettingReliableResetEnabled,            "[sett] ReliableResetEnabled   = %hhu", Settings->ReliableResetEnabled);
```

Comment:
Please make sure the spacing before the `=` aligns with all the others.

Reply:
Added a few spaces align =